sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -890,7 +890,7 @@ public void processElement(ProcessContext context) throws Exception {
         QueryResultBatch currentBatch = null;
 
         while (moreResults) {
-          Query.Builder queryBuilder = query.toBuilder().clone();
```

Comment:
Shouldn't dropping `clone` be enough since `toBuilder()` is short hand for effectively `newBuilder().mergeFrom(...)`?

Reply:
Good point- changed to toBuilder() in the next commit.